src/components/Main.tsx

```diff
@@ -0,0 +1,687 @@
+      screenOptions={{
+        drawerPosition: 'right',
+        drawerType: 'front',
+        drawerStyle: { backgroundColor: 'transparent', bottom: 0 },
+        headerShown: false
+      }}
+    >
+      <Drawer.Screen name="edgeAppStack" component={EdgeAppStack} />
+    </Drawer.Navigator>
+  )
+}
+
+const EdgeAppStack = () => {
+  const dispatch = useDispatch()
+
+  return (
+    <Stack.Navigator
+      initialRouteName="edgeTabs"
+      screenOptions={{
+        title: '',
+        headerTitle: ({ children }: HeaderTitleProps) => <HeaderTitle title={children} />,
+        headerLeft: () => <BackButton />,
+        headerRight: () => <SideMenuButton />,
+        headerShown: true,
+        headerTransparent: true
+      }}
+    >
+      <Stack.Screen
+        name="edgeTabs"
+        component={EdgeTabs}
+        options={{
+          headerShown: false
+        }}
+      />
+      <Stack.Screen name="createWalletSelectCrypto" component={ifLoggedIn(CreateWalletSelectCryptoScene)} />
+      <Stack.Screen name="createWalletSelectFiat" component={ifLoggedIn(CreateWalletSelectFiatScene)} />
+      <Stack.Screen
+        name="settingsOverview"
+        component={ifLoggedIn(SettingsScene)}
+        options={{
+          title: s.strings.title_settings
+        }}
+        listeners={{
+          focus: () => dispatch(showReEnableOtpModal())
+        }}
+      />
+      <Stack.Screen
+        name="transactionDetails"
+        component={ifLoggedIn(TransactionDetailsScene)}
+        options={{
+          headerTitle: () => <TransactionDetailsTitle />
+        }}
+        listeners={{
+          focus: async () => requestPermission('contacts')
+        }}
+      />
+
+      <Stack.Screen
+        name="createWalletImport"
+        component={ifLoggedIn(CreateWalletImportScene)}
+        options={{
+          headerRight: () => null
+        }}
+      />
+
+      <Stack.Screen
+        name="createWalletCompletion"
+        component={ifLoggedIn(CreateWalletCompletionScene)}
+        options={{
+          headerLeft: () => null,
+          headerRight: () => null
+        }}
+      />
+
+      <Stack.Screen
+        name="createWalletAccountSetup"
+        component={ifLoggedIn(CreateWalletAccountSetupScene)}
+        options={{
+          title: s.strings.create_wallet_create_account,
+          headerRight: () => <HeaderTextButton type="help" placement="right" />
+        }}
+      />
+
+      <Stack.Screen
+        name="createWalletAccountSelect"
+        component={ifLoggedIn(CreateWalletAccountSelectScene)}
+        options={{
+          title: s.strings.create_wallet_account_activate,
+          headerRight: () => <HeaderTextButton type="help" placement="right" />
+        }}
+      />
+
+      <Stack.Screen
+        name="transactionList"
+        component={ifLoggedIn(TransactionList)}
+        listeners={{
+          focus: async () => requestPermission('contacts')
+        }}
+        options={{
+          title: ' '
```

Comment:
Single space? A blank string is the default already.

Reply:
This was copied over from Main.ui.tsx.

Reply:
I removed it.